tests/Unit/Http/HttpClient.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace LaraCrafts\UrlShortener\Tests\Unit\Http;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Middleware;
+use Psr\Http\Message\ResponseInterface;
+use function GuzzleHttp\choose_handler;
+
+class HttpClient extends Client
```

Comment:
Instead of `HttpClient` it might be better to name this `HttpClientMock` or something more talkative.